llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -609,7 +646,8 @@ static TableFiles processOneModule(std::unique_ptr<Module> M, bool IsEsimd,
   {
     ImagePropSaveInfo ImgPSInfo = {
         true,          DoSpecConst,         SetSpecConstAtRT,
-        SpecConstsMet, EmitKernelParamInfo, IsEsimd};
+        SpecConstsMet, EmitKernelParamInfo, IsEsimd,
+        true};
```

Comment:
What is the point of passing `true` here? Why can't we just unconditionally search for asserts if we anyway pass `true` here?

Reply:
It looked that it's better to add an option to ImagePropSaveInfo - so we can understand which properties can be dumped. But now it seems useless, I agree with that. Do you think we should remove `IsAssertEnabled` option?

Reply:
It seemed it's better to add an option to ImagePropSaveInfo - so we can understand which properties can be dumped. But now it seems useless, I agree with that. Do you think we should remove the option?

Reply:
> Do you think we should remove `IsAssertEnabled` option?

Unless we provide a control for it (like command line option similar to spec constants or ESIMD lowering) I don't see a point in having this `ImagePropSaveInfo` field.